lib/cisco_node_utils/cmd_ref/interface.yaml

```diff
@@ -59,11 +67,18 @@ ipv4_netmask_length:
   default_value: ~
 
 ipv4_proxy_arp:
-  config_get_token_append: '/^ip proxy-arp$/'
-  config_set_append: "%s ip proxy-arp"
   default_value: false
+  nxapi:
+    config_get_token_append: '/^ip proxy-arp$/'
+    config_set_append: "%s ip proxy-arp"
+  grpc:
+    config_get_token_append: '/^proxy-arp$/'
+    config_set_append: "%s proxy-arp"
 
 ipv4_redirects_loopback:
+  test_config_result:
+    false: RuntimeError
+    true: RuntimeError
```

Comment:
Yup, same as in base code - this section is only used in a negative test for interface types that don't support ipv4 redirects as a configurable option. Kinda obtuse.